embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/internal/spans/CurrentSessionSpanImpl.kt

```diff
@@ -107,15 +107,21 @@ internal class CurrentSessionSpanImpl(
     private fun startSessionSpan(startTimeMs: Long): EmbraceSpan {
         traceCount.set(0)
 
+        val spanName = "session".toEmbraceSpanName()
```

Comment:
Had to move the logic to prepend `emb` outside side of the SpanBuilder because we need to access it for snapshot purposes, AND we also need to pass it in as a parameter to the friggin builder. I hate this, but not much we can do. The good thing is that this is contained - the logic is here and in SpanServiceImpl.